### ISSUE TITLE
Anaconda licensing issues

### ISSUE THREAD

**Issue by marklcrowe** [swcarpentry/python-novice-gapminder#620]
<details>
<summary><strong>Anaconda may not be freely available to trainees</strong></summary>

Hi all, I've just had a message from a trainee on an upcoming workshop that he's not able to use Anaconda Navigator to install Python/Jupyter because Anaconda have become much more restrictive in their licencing conditions recently. He works at a not for profit medical research institute, but because they don't grant degrees, Anaconda doesn't recognise them as 'academic'. That means they want a commercial licence fee, which the organisation can't afford, so none of their researchers can use Anaconda. 

More generally, this would suggest that many potential trainees wanting to attend Python training may not be able to access Anaconda, or may have to pay (in some cases quite a significant) licence fee to do so. Should the Carpentries continue to recommend a commercial product that costs our learners money and so reduces equity and accessibility for our training, or can we look into alternatives (in this particular case, they hope to use miniconda).

</details>

**Comment by frfeng** [comment]
I agree. Anaconda is a good one-stop shop, but it has its limitations in education as Mark mentioned above. An alternative could be to install Python and JupyterLab directly, and pip install the key packages?

**Comment by vahtras** [comment]
What about miniconda, that must still be free. One would only need a
configuration file (yaml) which defines all the necessary packages?

**Comment by alee** [comment]
I think we might want to refer this to the Curriculum Advisory Committee as this is a larger issue that affects all the python based carpentries lessons.

I don't know all the details here but this SO question has some additional context:

https://stackoverflow.com/a/74762864/93370

**Comment by bsmith89** [comment]
Thanks for reaching out! I've opened a parallel issue in swcarpentry/curriculum-advisors#10 where the committee members will discuss this topic. You are welcome to contribute additional context and important information there as well.

**Comment by sstevens2** [comment]
I think this issue can be closed with the updated install directions to use conda-forge instead.

**Comment by alee** [comment]
thanks @sstevens2 !

still need to resolve #646 and remove references to Anaconda in the lesson's first page.
